apps/www/_blog/2024-11-30-hack-the-base.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title: 'Hack the Base! with Supabase'
+description: 'Play cool games, win cool prizes.'
+author: stephen_morgan
+launchweek: '13'
+categories:
+  - launch-week
+tags:
+  - launch-week
+  - security
+date: '2024-12-02'
+---
+
+## Are you ready to test your hacking skills?
+
+Supabase is excited to announce our first-ever Capture the Flag (CTF) challenge, [**Hack the Base**](https://ctf.supabase.com). Whether you're a seasoned hacker or just starting out, this challenge is for you.
```

Comment:
Don't talk about Supabase in the 3rd person - you can say "We are "...

Also, I'd keep emotion out of it. Don't say things like "excited". Just say: "we're announcing our first Capture the Flag .."

Reply:
one more thing - move the abreviation to the next paragraph - Capture the Flag (CTF)

Your first paragraph should be clear and succinct